calcom/cal.com
# Ensure currentUsername is set before comparing it with username

## What does this PR do?



Fixes #4749



**Environment**: Production

## Type of change





## How should this be tested?



- Impersonate a user and refresh multiple times the profile page to see the issue.
## Checklist


- I haven't added tests that prove my fix is effective or that my feature works
